app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/directory/ContactsCreationTest.kt

```diff
@@ -156,7 +160,9 @@ class ContactsCreationTest {
         notesEditText.perform(replaceText("This is a test note."))
 
         onView(withId(R.id.confirm_form)).perform(click())
-        Thread.sleep(3000)
```

Comment:
I think I put the delay here, you should be able to take it off, it was for debugging purposes.